crates/conductor_lib/src/keystore.rs

```diff
@@ -119,8 +119,11 @@ impl Keystore {
     /// This will query `passphrase_manager` immediately to set a passphrase for the keystore.
     pub fn new(
         passphrase_manager: Arc<PassphraseManager>,
-        hash_config: Option<PwHashConfig>,
+        mut hash_config: Option<PwHashConfig>,
     ) -> HcResult<Self> {
+        if hash_config.is_none() {
+            hash_config = test_hash_config()
+        }
```

Comment:
I feel like we could just get rid of this line of code? Seems to be doing the same thing line 163 is doing. That way hash_config doesn't have to be mutable as well